package.json

```diff
@@ -21,6 +21,9 @@
 , "contributors": [
     { "name":   "Jann Horn"
     , "github": "thejh"
+    }, 
+	{ "name":   "Markus Sattler"
```

Comment:
Indentation: Looks like another tab. Please use grep or something to look for others...